lib/config.py

```diff
@@ -36,6 +36,15 @@
         'username': 'OpenStack user name with Swift privileges',
         'password': 'OpenStack user password'
     },
+    'credhub': {
+        'credhub_url' : 'Credhub endpoint',
+        'credhub_uaa_url': 'Endpoint of UAA for credhub',
+        'credhub_key': 'key to the iaas credentials object in credhub',
+        'credhub_client_id': 'OAUTH client id used in access token generation',
```

Comment:
shouldn't all the credentials used for UAA be named as uaa_ instead of credhub_

Reply:
i thought of the same, but since these UAA users are going to be mainly used from credhub service prefixed it the same so as to know. i.e. to access credhub what is the user id / pwd i would need and hence the naming.